### ISSUE TITLE
This plugin is missing some stuff

### ISSUE THREAD

**Issue by highstrike** [dereuromark/cakephp-test-helper#4]
You must include the assets for the "backend" interface found in `/test-helper`

This plugin repository is currently lacking:
- it's own layout.ctp.
- assets like jquery, font awesome and the modal lib that are being used in the templates.
- the Flash component should also be loaded by default as it's being used in the controllers.
- the Flash design elements (`Element/Flash/success.ctp` and `Element/Flash/error.ctp`) maybe look [at my component](https://github.com/unimatrix/backend/blob/master/src/Controller/Component/FlashComponent.php) and include that in the project as well because it will load the elements from the plugin (once included).
- the [Tools.Format](https://github.com/dereuromark/cakephp-tools/blob/master/src/View/Helper/FormatHelper.php) helper should also be loaded by default as it's being used in the views.
- the [Tests](https://github.com/dereuromark/cakephp-test-helper/blob/master/src/View/Helper/TestHelperHelper.php) helper should also be loaded by default

## Extra stuff needed to make this plugin work

currently my `AppController.php` looks like this:
```
class AppController extends Controller
{
    public function initialize() {
        parent::initialize();

        // dereuromark's testhelper
        if($this->request->getParam('plugin') == 'TestHelper') {
            $this->loadComponent('Flash');
        }
    }
}
```

and my `AppView.php` looks like this:
```
class AppView extends View
{
    public function initialize() {
        parent::initialize();

        // dereuromark's testhelper
        if($this->request->getParam('plugin') == 'TestHelper') {
            $this->loadHelper('Test');
            $this->loadHelper('Tools.Format');
        }
    }
}
```

**Comment by dereuromark** [comment]
Anything still missing?
Maybe we can draw up a PR?

**Comment by ravage84** [comment]
I Agree, the plugin (at least for CakePHP 3.x) should come with a theme comtaining a default layout of its own with everything needed to display a workable UI. Could be vanilla Bootstrap or something more simple. Kinda like "Batteries included".

**Comment by highstrike** [comment]
I never stopped beliving in you @dereuromark 😂
